projects/lib/src/board/westernboard.h

```diff
@@ -76,6 +76,29 @@ class LIB_EXPORT WesternBoard : public Board
 		/*! Movement mask for Rook moves. */
 		static const unsigned RookMovement = 8;
 
+		/*! Types of Pawn moves. */
+		enum StepType: const char
+		{
+			 Free = (char) 1,      //!< May move if target is empty
+			 Capture = (char) 2,   //!< Capture opposing piece only
+			 /* FreeOrCapture = Free|Capture, //!< like King or Sergeant */
+			 None =(char) 0        //!< Cannot move here
+		} stepTypeEnum;
+
+		/*!
+		 * Movement mask for Pawn moves.
+		 * Forward: Left diagonal, straight, right diagonal.
+		 * \sa BerolinaBoard
+		 *
+		 * Default: A pawn can step straight ahead onto a free square or
+		 * capture diagonally forward.
+		 */
+		struct PawnStep { StepType type; int file; };
+		QList<PawnStep>  pawnSteps = { {Capture, -1}, {Free, 0}, {Capture, 1} }; 
```

Comment:
Well, I deliberately put it in the declaration part to have the default assignment close to the declaration of StepType for overview. I will change this.